code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -392,6 +392,7 @@
 		list("M3 Pattern Skull Marine Armor", 20, /obj/item/clothing/suit/storage/marine/skull, VENDOR_ITEM_REGULAR),
 		list("M3-EOD Pattern Heavy Armor", 10, /obj/item/clothing/suit/storage/marine/heavy, VENDOR_ITEM_REGULAR),
 		list("M3-L Pattern Light Armor", 10, /obj/item/clothing/suit/storage/marine/light, VENDOR_ITEM_REGULAR),
+		list("M3-VL Pattern Ballistics Vest", round(scale*1.5), /obj/item/clothing/suit/storage/marine/light/vest, VENDOR_ITEM_REGULAR),
```

Comment:
That won't work. Scale is only usable in `populate_product_list` and is mainly for balance. If you jsut want a static amount of things in the vendor like above lines, you can just put in the number.

Reply:
Oh fuck. Okay. Darn.